articles/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh.md

```diff
@@ -168,9 +164,7 @@ namespace TestConsole
 }
 ```
 
-The [SetDirty](/dotnet/api/microsoft.extensions.configuration.azureappconfiguration.iconfigurationrefresher.setdirty) method is used to set the cached value for key-values registered for refresh as dirty. This ensures that the next call to `RefreshAsync` or `TryRefreshAsync` re-validates the cached values with App Configuration and updates them if needed.
-
-A random delay is added before the cached value is marked as dirty to reduce potential throttling in case multiple instances refresh at the same time. The default maximum delay before the cached value is marked as dirty is 30 seconds, but can be overridden by passing an optional `TimeSpan` parameter to the `SetDirty` method.
+Ensuring that users get their most up to date configurations is nearly impossible as a configuration can change at any moment (even after a request is sent out). The `ProcessPushNotification` method incorporates a synchronization token into the next request to the `App Configuration`. This `Synchronization token` guarantees users receive all changes at least up to the point which the notification was triggered. There still is no guarantee that changes occuring after a push notification was triggered will be reflected in the response.
 
 > [!NOTE]
 > To reduce the number of requests to App Configuration when using push refresh, it is important to call `SetCacheExpiration(TimeSpan cacheExpiration)` with an appropriate value of `cacheExpiration` parameter. This controls the cache expiration time for pull refresh and can be used as a safety net in case there is an issue with the Event subscription or the Service Bus subscription. The recommended value is `TimeSpan.FromDays(30)`.
```

Comment:
Can we remove "The recommended value is `TimeSpan.FromDays(30)`."? This can vary a lot depending on user scenarios. I don't think we recommend 30 days.

Reply:
BTW, can you please update the code `.SetCacheExpiration(TimeSpan.FromDays(30))  // Important: Reduce poll frequency` to 1 day/24 hours. 30-day is a bit extreme.